### ISSUE TITLE
memory leak when using capture.transformed_ir

### ISSUE THREAD

**Issue by charlesTbot** [etiennedub/pyk4a#123]
Memory usage increased significantly while using capture.transformed_ir repeatedly.

```
from pyk4a import PyK4A
import os, psutil # for memory usage tracking

process = psutil.Process(os.getpid())

k4a = PyK4A()
k4a.start()

i = 0
print(process.memory_info().rss)

while i < 100:
    capture = k4a.get_capture()
    if capture.transformed_ir is not None:
        i += 1

k4a.stop()
print(process.memory_info().rss)
```

**Comment by lpasselin** [comment]
Good catch, let's fix this.

**Comment by lpasselin** [comment]
The fix is in version 1.2.4 which is now available on pypi. Thank you for reporting this mistake!
